src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

```diff
@@ -430,6 +430,10 @@
                 <source>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</source>
                 <target>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</target>
             </trans-unit>
+            <trans-unit id="111">
+                <source>The detected encoding "{{ detected }}" does not match one of the accepted encoding: "{{ encodings }}".</source>
```

Comment:
I wonder if we should change the text to something like this (to be more inline with other messages):

```suggestion
                <source>The detected encoding {{ detected }} is invalid. Allowed encodings are {{ encodings }}.</source>
```

This would require an update of the `Charset` constraint on the `7.1` branch first.

Reply:
Yes, let's do that.

Reply:
I think we should be a bit more precise on what we mean by "encoding". Maybe use "character encoding" in the first sentence at least?

```suggestion
                <source>The detected character encoding {{ detected }} is invalid. Allowed encodings are {{ encodings }}.</source>
```

Reply:
I wouldn't remove quotes because CharsetValidator implodes encodings with double quotes, which individually quotes all charsets in the sentence.

Reply:
I agree with @xabbuh. What about removing the quotes in CharsetValidator?

Reply:
Alright, why not, it would actually be consistent with some other messages. What about this? 

```xml
<trans-unit id="110">
    <source>The extension of the file is invalid ({{ extension }}). Allowed extensions are {{ extensions }}.</source>
    <target>L'extension du fichier est invalide ({{ extension }}). Les extensions autorisées sont {{ extensions }}.</target>
</trans-unit>
<trans-unit id="111">
    <source>The detected character encoding is invalid ({{ detected }}). Allowed encodings are {{ encodings }}.</source>
    <target>L'encodage de caractères détecté est invalide ({{ detected }}). Les encodages autorisés sont {{ encodings }}.</target>
</trans-unit>
```

Using parentheses to match what's been done with translation 110.

Reply:
Prepared #53256 for this :slightly_smiling_face: